include/utility/vpd_specific_utility.hpp

```diff
@@ -217,127 +217,117 @@ inline std::string
                             const types::VPDMapVariant& parsedVpdMap)
 {
     auto expanded{unexpandedLocationCode};
-    if (auto ipzVpdMap = std::get_if<types::IPZVpdMap>(&parsedVpdMap))
+
```

Comment:
All the checks has been removed. How you can be sure that it will work for all types?

Reply:
Will revisit.
Issue
https://github.com/ibm-openbmc/openpower-vpd-parser/issues/474